docs/notary.md

```diff
@@ -124,19 +124,19 @@ construct and send the payload.
 
 Node module with the designated key monitors the network for `P2PNotaryRequest`
 payloads. It maintains a list of current requests grouped by main transaction
-hash, when it receives enough requests to correctly construct all transaction
-witnesses it does so, adds a witness of its own (for Notary contract witness) and
+hash. When it receives enough requests to correctly construct all transaction
+witnesses, it does so, adds a witness of its own (for Notary contract witness) and
 sends the resulting transaction to the network.
 
 If the main transaction with all witnesses attached still can't be validated
-because of fee (or other) issues, the node waits for `NotValidBefore` block of
+due to any fee (or other) issues, the node waits for `NotValidBefore` block of
 the fallback transaction to be persisted.
 
 If `NotValidBefore` block is persisted and there are still some signatures
 missing (or the resulting transaction is invalid), the module sends all the
 associated fallback transactions for the main transaction.
 
-After processing service request is deleted from the module.
+After, processing service request is deleted from the module.
```

Comment:
Not really appropriate, the meaning is that the service request is deleted after processing. Maybe needs to be rephrased a little.